rebuild django-build command in Makefile and fix some syntax error

Channing88 opened the pull request:
1. Rebuild django-build command in Makefile
2. Fix some syntax error in Makefile

enveng-group reviewed (changes requested):
Thank you @Channing88 for these excellent Makefile enhancements! The additions will make our development workflows much more efficient.

Before we merge, please address these minor issues:

1. Fix the typo in `.PHONY`: `update-update_recurring-inspection-dates` should be `update-recurring-dates`
2. Remove the duplicate `make venv` entry at the end of the help command

Once these items are addressed, I'll be happy to approve and merge the PR.

enveng-group reviewed (changes requested):
Thank you @Channing88 for these excellent Makefile enhancements! The additions will make our development workflows much more efficient.

Before we merge, please address these minor issues:

1. Fix the typo in `.PHONY`: `update-update_recurring-inspection-dates` should be `update-recurring-dates`
2. Remove the duplicate `make venv` entry at the end of the help command

Once these items are addressed, I'll be happy to approve and merge the PR.

Channing88 commented:
Hi @enveng-group,

I have fixed the requested changes:
- Corrected the typo in `.PHONY`
- Removed the duplicate `make venv` entry from the help command

Please kindly review again when available. Thanks!

enveng-group commented:
The changes have been merged into the staging branch from integration/v0.0.6. Closing this PR.
